Dockerfile

```
@@ -34,8 +34,7 @@ RUN ["make", "manager"]
 
 FROM debian:bullseye-slim
 
-RUN ["apt", "update"]
-RUN ["apt", "install", "-y", "ca-certificates"]
+RUN ["bash", "-c", "apt-get update && apt-get install -y ca-certificates"]
```

Thank you for the link. It makes sense indeed.
I think we can use the shell form here:
```suggestion
RUN apt update && apt install -y ca-certificates
```

---

Just in this one place or everywhere?

---

I usually try to only use the shell form when there is no other way 🙂

---

Changed, PTAL